_episodes/docker-desktop.md

```diff
@@ -0,0 +1,411 @@
+If we go back to the 'Images' tab and run `hello world` again, we'll see a new container appear.
+
+## Interacting with containers
+
+Not all containers are as short lived as the one we've been using.
+If we run the `docker/getting-started` image that we had pulled earlier, we will see something different happen.
+You can immediately notice the status under the container name is 'RUNNING' now.
+The 'Logs' tab is not too informative, but the 'Inspect' tab already shows more information.
+A process called 'NginX' is running.
+The 'Terminal' and 'Stats' tabs changed the most.
+Since the container is still running, the stats get shown, and we are able to launch a terminal *inside* the container.
+
+{::options parse_block_html="true" /}
+<div>
+<ul class="nav nav-tabs nav-justified" role="tablist">
+  <li role="presentation" class="active">
+    <a href="#gs_Logs" aria-controls="gs_Logs" role="tab" data-toggle="tab">Logs</a>  </li>
+  <li role="presentation">
+    <a href="#gs_Inspect" aria-controls="gs_Inspect" role="tab" data-toggle="tab">Inspect</a>  </li>
+  <li role="presentation">
+    <a href="#gs_Terminal" aria-controls="gs_Terminal" role="tab" data-toggle="tab">Terminal</a>  </li>
+  <li role="presentation">
+    <a href="#gs_Stats" aria-controls="gs_Stats" role="tab" data-toggle="tab">Stats</a>  </li>
+</ul>
+<div class="tab-content">
+  <article role="tabpanel" class="tab-pane active" id="gs_Logs">
+  ![Logs tab in container from getting-started image.](../fig/docker-desktop/18_gettingstarted_log.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_Inspect">
+  ![Inspect tab in container from getting-started image.](../fig/docker-desktop/19_gettingstarted_inspect.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_Terminal">
+  ![Terminal tab in container from getting-started image.](../fig/docker-desktop/20_gettingstarted_terminal.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_Stats">
+  ![Stats tab in container from getting-started image.](../fig/docker-desktop/21_gettingstarted_stats.png)
+  </article>
+</div>
+</div>
+<br/>
+{::options parse_block_html="false" /}
+
+Before trying to do anything in the terminal, let's look at the container list by clicking on the 'Containers' tab on the left.
+You'll see the green icon of the container indicating that it is still live, and indication of how long it's been running for.
+![Containers list, getting-started still running.](../fig/docker-desktop/22_gettingstarted_containers.png)
+
+Clicking on the container name again will take us back to the 'Logs' tab in the container.
+Lets try and interact with the terminal inside the container.
+If you print the working directory with `pwd` you'll get the base directory: `\`.
+You can also list the contents with `ls`, and the `docker-entrypoint` files are a dead giveaway that we are inside the container.
```

Comment:
personally would prefer "cat /etc/os-release"

Reply:
that would work on Alpine, but not in all systems, would it? checking for docker-entrypoint seemed simpler, it does not require explaining what the /etc/os-release is doing...